Proofs/Experiments/Abs.lean

```diff
@@ -29,15 +38,29 @@ theorem correct
   (h_s0_program : s0.program = program)
   (h_s0_err : read_err s0 = StateError.None)
   (h_s0_sp : CheckSPAlignment s0)
-  (h_run : sf = run program.length s0) :
+  (h_run : sf = run (program.length - 1) s0) :
```

Comment:
```suggestion
  -- (FIXME) We do not account for the effects of `ret` yet, so 
  -- we simulate `program.length - 1` number of instructions only.
  (h_run : sf = run (program.length - 1) s0) :
```

Reply:
Though... I believe for this particular program, since we don't see any SP action, we could try to simulate `ret` too. Want to try, @alexkeizer?

Reply:
I thought the problem with `ret` is that makes it hard to cleanly define the exit state, since the PC becomes symbolic after execit? Then again, we don't use VCG here, so I guess that won't be a problem in this specific instance

Reply:
It was in fact easy to run the whole program, I've updated the proof